src/big/big_int.cr

```diff
@@ -116,15 +116,16 @@ struct BigInt < Int
   end
 
   # a.add(b), a += b, avoiding allocations by mutating a
-  #
-  #    a = 100.to_big_i
-  #    a.add(10) => # a = 110
+  # ```
```

Comment:
Please, add an empty line between the comment and the code block, ditto all below.

Reply:
done